presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/IOPlanPrinter.java

```diff
@@ -517,6 +519,7 @@ private FormattedDomain parseDomain(Domain domain)
                     discreteValues -> formattedRanges.addAll(
                             discreteValues.getValues().stream()
                                     .map(value -> getVarcharValue(type, value))
+                                    .filter(Objects::nonNull)
```

Comment:
what's the purpose of this filter?

Reply:
ohh - Thank you.  @bhhari  requested that .getVarcharValue() should throw as it previously did so this filter is no longer necessary.  I had .getVarcharValue() returning null for unknown types and then dropping those here to provide some type of best effort Explain IO.  I can remove the filter.